README.md

```diff
@@ -42,10 +46,27 @@ Usage of ./server:
     	print version number and build date
 ```
 
-If you are working on the sqlite migrations, html templates or website assets, build the server with `go build -tags dev`.
-This way it won't use the assets that are embedded in the binary but read them directly from the local filesystem.
+If you want to view the development server in your browser:
+```sh
+# change to the root of the project (e.g. cd go-ssb-room) and generate the frontend's styling; requires npm
+go generate -tags dev ./...
```

Comment:
```suggestion
cd web/styles && go generate -tags dev
```

Reply:
the original would work as well right? or would that cause extra work? not sure if we have generate statements elsewhere.

if the original would work i think it's better, just cause it's like "whatever just run the thing where it's needed" and you stay in the same directory

Reply:
I think it could be debated if we want `go generate` here and not just document whats going on, which is:

```bash
cd web/styles
npm ci
npm run compile-dev
```

This why we remove the confusing `go generate` indirection that's not required for anything else anymore.

ps: people also need to know that they need to run this before building the server.


```bash
cd web/styles
npm ci
npm run compile-prod
```
(See #52 for some discussion about this)

Reply:
yup up to you, i think we should merge this and you can change it to what you want, separately :)

i never really liked the concept of debate class ;)

Reply:
Fine by me :) It definitely works the way you wrote it.